projects/storefrontlib/src/cms-components/checkout/components/shipping-address/address-form/address-form.component.spec.ts

```diff
@@ -318,12 +311,6 @@ describe('AddressFormComponent', () => {
     );
   });
 
-  it('should call verifyAddress() when address has some changes', () => {
```

Comment:
Why these 2 tests were removed?

Reply:
They were temporarily removed. They're back now and improved.